custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -347,23 +347,23 @@
             "type": "SEPARATOR"
         },
         {
-            "name": "case_insensitive",
-            "label": "Case-insensitive",
+            "name": "case_insensitivity",
+            "label": "Case-insensitivity",
             "description": "Match keywords whatever the case (upper or lower)",
             "type": "BOOLEAN",
             "defaultValue": false
         },
         {
-            "name": "lemmatization",
-            "label": "Lemmatization",
-            "description": "Simplify words to their “lemma” form e.g. going → go, mice → mouse",
+            "name": "unicode_normalization",
+            "label": "Normalization",
+            "description": "Add Diacritic removal after NFD normalization",
             "type": "BOOLEAN",
             "defaultValue": false
         },
         {
-            "name": "unicode_normalization",
-            "label": "Unicode normalization",
-            "description": "Apply NFD Unicode normalization before matching",
+            "name": "lemmatization",
+            "label": "Lemmatization",
+            "description": "Simplify words to their “lemma” form e.g. going -> go, mice -> mouse",
```

Comment:
What happened to https://github.com/dataiku/dss-plugin-nlp-analysis/pull/7/files?

Reply:
e7e7116